com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/EvaluateWaterRefractionData.cs

```diff
@@ -0,0 +1,64 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class EvaluateWaterRefractionDataNode : IStandardNode
+    {
+        public static string Name => "EvaluateWaterRefractionData";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+@"ComputeWaterRefractionParams(positionWS, NormalWS, LowFrequencyNormalWS, screenPos.xy, viewWS, true, _MaxRefractionDistance, _TransparencyColor.xyz, _OutScatteringCoefficient, RefractedPositionWS, DistordedWaterNDC, refractedDistance, AbsorptionTint);",
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("refractedDistance", TYPE.Float, Usage.Local),
+                new ParameterDescriptor("positionWS", TYPE.Vec3, Usage.Local, REF.WorldSpace_Position),
+                new ParameterDescriptor("viewWS", TYPE.Vec3, Usage.Local, REF.WorldSpace_ViewDirection),
+                new ParameterDescriptor("screenPos", TYPE.Vec2, Usage.Local, REF.ScreenPosition),
+                new ParameterDescriptor("NormalWS", TYPE.Vec3, Usage.In),
+                new ParameterDescriptor("LowFrequencyNormalWS", TYPE.Vec3, Usage.In),
+                new ParameterDescriptor("RefractedPositionWS", TYPE.Vec3, Usage.Out),
+                new ParameterDescriptor("DistordedWaterNDC", TYPE.Vec2, Usage.Out),
```

Comment:
This is spelled wrong. It should be DistortedWaterNDC. Distort is spelled with a T. This was also wrong in SG1, so it's not your fault. Remember to fix it in the ParameterUIDescriptor and body code also.